spec/mongo/crypt/binding/binary_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'mongo'
+require 'support/lite_constraints'
+
+RSpec.configure do |config|
+  config.extend(LiteConstraints)
+end
+
+describe 'Mongo::Crypt::Binding' do
+  describe 'binary_t bindings' do
```

Comment:
These were moved from the `spec/mongo/crypt/binding_spec` file because it was getting too big to read easily.